code_challenges.rb

```diff
@@ -2,48 +2,61 @@
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest. Use the test variables provided.
-
 full_arr1 = [4, 9, 0, '7', 8, true, 'hey', 7, 199, -9, false, 'hola']
 # Expected output: [-9, 7, 9, 199]
 full_arr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
-
-
-
-
-
+#Create a method that will take in an array and return one with only odd numbers.
+def odd_array array
+    new_array = array.select { |value| value.is_a?(Integer) && value % 2 != 0}
+    print new_array.sort
+end
+odd_array full_arr1
+odd_array full_arr2
```

Comment:
Looks good. As a workflow, I would recommend printing the method call rather than the inner working of the method. Meaning print line 14 and 15 rather than one inside the method.